test/boulder-tools/setup-govulncheck.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -feuxo pipefail
+
+# Implements a filesystem structure that can serve as an API for govulncheck.
+# See https://go.dev/security/vuln/database for more information.
+#
+# TODO(@pgporada) Remove this if we decide that the boulder-tools container can
+# have network access.
+
+GVCDIR="/tmp/govulncheck-local-database"
+if [ -d "${GVCDIR}" ]; then
+    rm -rf "${GVCDIR}"
+fi
+
+mkdir -p "${GVCDIR}"
+cd "${GVCDIR}"
+
+# Check out only the data we need from this repository.
+git clone -n \
```

Comment:
Did you try running this as just:
1) clone the whole repo (maybe with depth=1) to some throwaway path
2) mv the data/osv directory from there to /tmp/govulncheck-local-database/ID
3) delete the throwaway directory?

I honestly don't have a good sense for how big this vulndb repo is, and whether doing the sparse checkout makes things notably faster. If it does, awesome! If it doesn't, I'd consider this other simpler approach just so others can read/maintain/modify this script without having to know the guts of git quite as well.